metadata.json

```diff
@@ -18,7 +18,7 @@
     },
     {
       "name": "puppetlabs/apache",
-      "version_requirement": ">= 8.0.0 < 9.0.0"
+      "version_requirement": ">= 9.0.0"
```

Comment:
Maybe we should cap it below 10 as well?
```suggestion
      "version_requirement": ">= 9.0.0 < 10.0.0"
```

Reply:
Wow, I got distracted :-) Indeed we should do that!

Reply:
Committed, thanks for review.